crates/settlement-clients/ethereum/Cargo.toml

```diff
@@ -4,18 +4,18 @@ version.workspace = true
 edition.workspace = true
 
 [dependencies]
-alloy = { workspace = true, features = ["full", "node-bindings"] }
 alloy-primitives = { version = "0.7.7", default-features = false }
+
+alloy = { workspace = true, features = ["full", "node-bindings"] }
```

Comment:
Wanted to update alloy to latest version, but it's specifically required with varied features in the crates. Left it as it is for now.
<img width="522" alt="Screenshot 2024-09-06 at 12 38 53 PM" src="https://github.com/user-attachments/assets/f2a8f955-d006-46a0-bc45-18411badf03e">

Reply:
are you sure, it's possible that we just copied the import from somewhere and maybe the both work on the latest version?